### ISSUE TITLE
Brackets on Mac crashes when launched with Zscaler turned on.

### ISSUE THREAD

**Issue by jkerr007** [adobe/brackets#15168]
### Prerequisites

* [ ] Can you reproduce the problem with `Debug -> Reload Without Extensions`?
Yes
* [ ] Did you perform a cursory search to see if your bug or enhancement is already reported?
Yes
* [ ] Did you read the [Troubleshooting guide](https://github.com/adobe/brackets/wiki/Troubleshooting)?
No

### Description
Brackets on Mac crashes when launched with Zscaler security settings turned on.
Could you please let me know if there are certain sites that Brackets reaches out to when this app launched? If so, please let me know so I can give this list to Zscaler.

### Steps to Reproduce
1.) Launch Brackets with Zscaler security settings on (this is on by default).
2.) Brackets eventually crashes.
3.) Turn off security settings in Zscaler, Brackets launches without issues.

**Expected behavior:** [What you expected to happen]
It should just open as expected.

**Actual behavior:** [What actually happened]
It crashes. There's a crash report about proxy resolver.

Process:               Brackets [29198]
Path:                  /Applications/Brackets.app/Contents/MacOS/Brackets
Identifier:            io.brackets.appshell
Version:               1.14.2 (1.14.2)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           Brackets [29198]
User ID:               501
Date/Time:             2020-06-04 16:23:53.569 -0400
OS Version:            Mac OS X 10.14.6 (18G5033)
Report Version:        12
Bridge OS Version:     3.0 (14Y906)
Anonymous UUID:        342F11FC-076E-8F4A-448B-5FB0F39FE650
Sleep/Wake UUID:       E87DA375-3F86-45B2-93BE-C17E503870EC
Time Awake Since Boot: 34000 seconds
Time Since Wake:       23000 seconds
System Integrity Protection: enabled
Crashed Thread:        22  Proxy Resolver
Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000000010
Exception Note:        EXC_CORPSE_NOTIFY
Termination Signal:    Segmentation fault: 11
Termination Reason:    Namespace SIGNAL, Code 0xb
Terminating Process:   exc handler [29198]

### Versions

Please include the OS and what version of the OS you're running.
MacOS Mojave version 10.14.6

Please include the version of Brackets. You can find it under `Help -> About Brackets` (Windows and Linux) or `Brackets -> About Brackets` (macOS)
1.14.2

**Comment by craftmash** [comment]
Same here. +1 to above issue. Same info except for UUID and process number.

**Comment by jkerr007** [comment]
Hey craftmash, just to clarify when you say experiencing the crash - do you have Zscaler running (i.e. with security settings on)?

**Comment by craftmash** [comment]
Yes it is active. I don't have a choice, it's an enforced work policy on this laptop. I can't turn it off. I rolled back to v1.14.1 and it works fine. Tried 1.14.2 again and it crashed on startup as before, so back to 1.14.1 for now.

**Comment by jkerr007** [comment]
Great! Glad to hear I am not the only getting this issue. I'm working with Zscaler support. However, they are needing info from Brackets developers on what is it reaching out to when it is getting launched.

Thanks for the tip on installing an older version of Brackets!

**Comment by nstrauss** [comment]
Getting crashes here whenever a system proxy is enabled. We use a product similar to Zscaler. Can any Brackets dev comment on what addresses are checked on launch? If so we could probably add to allow list.
